Flow.Launcher.Core/Plugin/JsonRPCPluginV2.cs

```diff
@@ -139,11 +140,24 @@ public virtual Task ReloadDataAsync()
             return Task.CompletedTask;
         }
 
-        public virtual ValueTask DisposeAsync()
+        public virtual async ValueTask DisposeAsync()
         {
+            try
+            {
+                await RPC.InvokeAsync("close");
+            }
+            catch (RemoteMethodNotFoundException e)
+            {
+            }
+            catch (Exception e)
+            {
+                Log.Exception(
```

Comment:
Is this the case where the plugin hasn't imemted the 'close' method? If so do we need to be logging it?

Reply:
This is not the case that plugin didn't implement the `close` method. That is the previous exception branch. I don't think we really need to log it, because if plugin doesn't have the close method, we can safely assume that they don't need it. Further, given that we already release jsonrpc v2 even though without documentation and as prerelease, it may be a problem to break the backward compatibility. And it doesn't gain us anything for logging it, unless we mark it as exception?